configs/KAKUTEF7V2/config.h

```diff
@@ -80,31 +82,29 @@
 #define USB_DETECT_PIN       PA8
 
 #define TIMER_PIN_MAPPING \
-    TIMER_PIN_MAP( 0, PE13, 1,  1) \
-    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 3, PE9 , 1,  2) \
-    TIMER_PIN_MAP( 4, PE11, 1,  1) \
-    TIMER_PIN_MAP( 5, PC9 , 2,  0) \
-    TIMER_PIN_MAP( 6, PA3 , 2,  0) \
-    TIMER_PIN_MAP( 7, PD12, 1,  0)
-
+    TIMER_PIN_MAP( 0, RX_PPM_PIN        , 1,  1) \
```

Comment:
```suggestion
    TIMER_PIN_MAP( 0, RX_PPM_PIN        , 1,  -1) \
```
Remove RX_PPM as it conflicts with timer for Motors 3 and 4

Reply:
Done above. Please refresh.